src/views/Pools/components/CakeVaultCard/index.tsx

```diff
@@ -48,60 +48,67 @@ const CakeVaultCard: React.FC<CakeVaultProps> = ({ pool, showStakedOnly }) => {
   }
 
   return (
-    <StyledCard isStaking={accountHasSharesStaked}>
-      <StyledCardHeader isAutoVault earningTokenSymbol="CAKE" stakingTokenSymbol="CAKE" />
-      <StyledCardBody isLoading={isLoading}>
-        <AprRow
-          pool={pool}
-          stakingTokenPrice={stakingTokenPrice}
+    <StyledCard isPromotedPool>
+      <StyledCardInner>
```

Comment:
The addition of this element has sent the diff a bit haywire. Very little has changed here